vuepress/docs/next/docs/getting-started/development-concepts.md

```diff
@@ -0,0 +1,59 @@
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+
+	
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means development teams can bundle and share components within a central catalog. Multiple teams or groups can  collaborate on single components or whole packaged business capability (PBC) across organizations and applications. They can test and assess components for readiness, manage and publish versions and share metadata for continuous upgrades. 
+
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Bundle and Component Descriptors](../curate/ecr-bundle-details.md) | [Install Bundles from a Private Git Repository](../../tutorials/curate/ecr-private-git-repo.md) |
+| [Filtering Bundles](../curate/ecr-bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../../tutorials/curate/ecr-private-images.md) |
+| [Customize Bundle Info in App Builder](../curate/ecr-bundle-presentation-config.md) |   |
+| [Bundle Version and Updates FAQ](../curate/ecr-bundle-versions-faq.md) |  |
+| [Bundle Uninstall](../curate/ecr-uninstall-flow.md) |  |
+
+
+## Compose an Application 
+The Compose stage on Entando is centered around the [Entando App Builder](../compose/app-builder.md), a low-code composition user interface. Within the App Builder, application owners can assemble a new component, update an app, or add templates, widgets or fully functional PBCs. The [Entando Component Manager](../compose/ecm-overview.md) and [Entando Component Repository](../compose/ecr-overview.md) are integrated into the App Builder, making the composing process flexible and agile by managing all your components there.
+
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Welcome Wizard](../compose/welcome-wizard.md) | [Page Management](../../tutorials/compose/page-management.md) |
+| [Entando Component Repository](../compose/ecr-overview.md) | [Widgets and Fragments](../../tutorials/compose/widgets-fragments.md) |
+| [Entando App Builder](../compose/app-builder.md) | [Content Creation](../../tutorials/compose/content-tutorial.md) |
+| [Entando Component Manager](../compose/ecm-overview.md) | [Content Templates](../../tutorials/compose/content-templates-tutorial.md)  |
+|  | [Digital Assets](../../tutorials/compose/digital-assets-tutorial.md) |
+
+## Consume Applications 
+In the development lifecycle of an application, Consume is the execution stage where end users interact with the app to complete certain tasks. For an Entando application, the modular design of the components and services means each can be scaled individually or as a whole, according to those interactions. Likewise, content can be added or changed, engine rules adapted and upgrades made seamlessly. Thus the development cycle is a continuous process with minimal disruptions and faster time to market.
```

Comment:
"In" an Entando Application, not "for"

Entando Application -> capitalize

same comment wrt "means"
tagging ".., according to those interactions" on the end is awkward

what are upgrades outside of adding/editing content and adapting engine rules? if upgrades fall within those categories mentioning upgrades is redundant. seamless upgrades comprise improvements to content and engine rules 

use of "thus" is awkward